src/commands/scanner/run.ts

```diff
@@ -219,7 +220,7 @@ export default class Run extends ScannerCommand {
 	}
 
 	private deriveFormatFromOutfile(): OUTPUT_FORMAT {
-		const outfile = this.flags.outfile;
+		const outfile = this.flags.outfile as string;
```

Comment:
Same as above here. Can we the type while defining the const?

Reply:
Same as above, we need to type `this.flags.outfile` to satisfy `@typescript-eslint/no-unsafe-assignment`.